omod/src/main/java/org/openmrs/module/sync2/page/controller/Sync2PageController.java

```diff
@@ -0,0 +1,30 @@
+package org.openmrs.module.sync2.page.controller;
+
+import org.apache.commons.lang.StringUtils;
+import org.openmrs.module.sync2.api.utils.SyncUtils;
+import org.openmrs.module.uicommons.util.InfoErrorMessageUtil;
+import org.openmrs.ui.framework.UiUtils;
+import org.openmrs.ui.framework.page.PageModel;
+import org.springframework.stereotype.Controller;
+
+import javax.servlet.http.HttpSession;
+
+@Controller
+public class Sync2PageController {
```

Comment:
I agree that javadoc should be helpful, especially in open source projects. However, as far as I can see other controllers do not have javadocs. Nevertheless, I think it would be good to start adding javadocs to newly created public class/methods

Reply:
Okay, I added the Javadoc.